scripts/maint/updateFallbackDirs.py

```diff
@@ -1609,14 +1613,14 @@ def apply_filter_lists(self, whitelist_obj, exact=False):
       else:
           # exclude
           excluded_count += 1
-          log_excluded('Excluding %s: not in whitelist.',
-                       f._fpr)
+          log_excluded('Excluding %s: not in %s.',
+                       f._fpr, list_type)
     self.fallbacks = filtered_fallbacks
     return excluded_count
 
   @staticmethod
   def summarise_filters(initial_count, excluded_count):
-    return '/* Whitelist excluded %d of %d candidates. */'%(
+    return '/* Fallback list excluded %d of %d candidates. */'%(
```

Comment:
Please use check_existing here.

Reply:
Did so in 5952762901df4eee120f6155999ece4796909050.